imap/imapd.c

```diff
@@ -14859,12 +14862,16 @@ static void cmd_compress(char *tag, char *alg)
 static void cmd_enable(char *tag)
 {
     static struct buf arg;
-    int c;
+    int c = EOF;
     unsigned new_capa = 0;
 
-    /* RFC 5161 says that enable while selected is actually bogus,
-     * but it's no skin off our nose to support it, so don't
-     * bother checking */
+    if (imapd_index || backend_current) {
```

Comment:
Was this a hard requirement for IMAP4rev2?  We probably allowed this because some dodgy client was doing the wrong thing.

I think this is the one thing I'm concerned about maybe breaking somebody here.

Reply:
@brong I don't recall if allowing this was an issue with IMAP4rev2.  I'll do some testing and allow it if its not problematic

Reply:
Removed this check and reverted to allowing ENABLE while SELECTEd